### ISSUE TITLE
Typo in the Asynchronous Programming howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3267]
|<img alt="Wolf's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Wolf reported|
|-|-|
|Trac ID|trac#3267|
|Type|defect|
|Created|2008-06-01 21:31:32Z|

"This document is for readers new to Twisted who are familiar with the Python programming language and, at least conceptually, with core networking __conepts__ such as servers, clients and sockets." -> concepts

Attachments:

* [async.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/2dc/2dc0c690d3726aed41ab1c89d2ed65c9f01e4337/cc30476ded32b7ec7c8945ca1136c969f9357b5c.patch) (790 bytes) - added by Wolf on 2008-06-01 21:32:06Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3267 3267
type__defect defect
reporter__Wolf Wolf
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1212355892000000 1212355892000000
changetime__1212413511000000 1212413511000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun|
|-|-|

Looks good.  Will apply.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [23817]) Apply fix for spelling mistake

Author: Wolf
Reviewer: exarkun
Fixes: [#3267](https://github.com/twisted/twisted/issues/3267)

Correct a simple spelling error.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
